com.unity.render-pipelines.universal/Runtime/NormalReconstruction.cs

```diff
@@ -0,0 +1,41 @@
+namespace UnityEngine.Rendering.Universal
+{
+    /// <summary>
+    /// Util class for normal reconstruction.
+    /// </summary>
+    public static class NormalReconstruction
```

Comment:
Idk, you think it will not be useful for users? I could move it to internal namespace (It is used by our tests now)

Reply:
I think it can be. I am just concerned that users will expect a lot more functionality from a class called `NormalReconstruction`. I would expect it offers me to reconstruct normals, but I need quite a bit more to do this; NormalReconstructionTestFeature.cs and a material using NormalReconstruction.hlsl ?